.github/workflows/pullrequest.yml

```diff
@@ -6,22 +6,20 @@ concurrency:
 
 on:
   workflow_dispatch:
-  pull_request:
+  pull_request_target:
     branches: [ 'main' ]
-    types: [opened, synchronize, reopened]
+    types: [opened, synchronize, reopened, unlabeled]
   push:
     branches: [ '!main' ]
```

Comment:
Feel like these should be removed. Everything should go through a PR, right? And labels won't be a thing if it's a random branch and not a PR.

Reply:
removed in babf36dd